index.html

```diff
@@ -767,184 +675,155 @@ <h3 id="placement-of-jukugo-ruby">Placement of Jukugo-ruby</h3>
-"Japanese characters". 
+spacing is inserted between every character in the ruby base text 
+as well as at the start and the end of the ruby base text. 
+Following placement of the ruby base text, 
+place the shorter ruby annotations based on the length of the ruby base text without spacing at the start and the end 
+but with inter-character spacing when the ruby base text is 
+<a title="japanese characters">Japanese characters</a>. 
 </p>
 <p>
-When the length of the shorter ruby string is longer than the ruby base 
-character string with inter-character spacing, the shorter ruby string is 
-set solid and ruby string is placed so that its center match that of the 
-ruby base character string 
-(see <a href="#double-group-2"></a>).
-
-<figure id=double-group-2>
-  <img src="img/fig25.svg" />
-  <figcaption>Double-sided ruby example 2 with both group-ruby</figcaption>
+When the width of the shorter ruby annotation is longer than its ruby base 
+text with inter-character spacing, the shorter ruby annotation is 
+set solid and  is placed so that its center matches that of its base text (see [[[#double-group-2]]]).</p>
+
+
+<figure id="double-group-2">
+  <img src="img/fig25.svg" alt="" style="min-width: 12em;" />
+  <figcaption>Second example of double-sided ruby where both sides are group-ruby.</figcaption>
 </figure>
 
-</p>
 <p>
-When the length of the shorter ruby string is shorter than the ruby base 
-character string with inter-character spacing, 
-follow the rules for placement of group-ruby described in 
+When the width of the shorter ruby annotation is shorter than its base 
+text with inter-character spacing, 
+follow the rules for the placement of group-ruby described in 
 [[[#placement-of-group-ruby]]], 
-using the length of the ruby base character string with inter-character 
+using the length of the ruby base text with inter-character 
 spacing. 
-When the shorter ruby string is consisted with "Japanese characters" described in 
+When the shorter ruby string consists of <a title="japanese characters">Japanese characters</a>, as described in 
 [[[#placement-of-group-ruby]]],
-spacing is inserted between every character in the ruby string 
-as well as the start and the end of the ruby string 
-(see <a href="#double-group-3"></a>).
+spacing is inserted between every character in the ruby annotation 
+as well as at the start and the end of the ruby annotations (see [[[#double-group-3]]]).</p>
 
-<figure id=double-group-3>
-  <img src="img/fig26.svg" />
-  <figcaption>Double-sided ruby example 3 with both group-ruby</figcaption>
+<figure id="double-group-3">
+  <img src="img/fig26.svg" alt="" style="min-width: 12em;" />
+  <figcaption>Third example of double-sided ruby where both sides are group-ruby.</figcaption>
 </figure>
 
-</p>
+
 <p>
-For other points, follow the same rules for placement of mono-ruby 
+For other points, follow the same rules as for the placement of mono-ruby 
```

Comment:
Should "mono-ruby" be "group-ruby" here?

Reply:
I checked the Japanese original, and i agree that it should be 'group'.  Fixed.

Reply:
yes. (my mistake for English text)